packages/vue-vanilla/src/cells/TextCell.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <input
+    :id="control.id + '-input'"
+    :class="styles.control.input"
+    :value="control.data"
+    :disabled="!control.enabled"
+    :autofocus="appliedOptions.focus"
+    :placeholder="appliedOptions.placeholder"
+    @change="onChange"
+    @focus="isFocused = true"
+    @blur="isFocused = false"
+  />
+</template>
+
+<script setup lang="ts">
+import {
+  ControlElement,
+  isStringControl,
+  type RankedTester,
+  rankWith,
+} from '@jsonforms/core';
+import { rendererProps, useJsonFormsControl } from '@jsonforms/vue';
+import { useVanillaControl } from '../util';
+
+const props = defineProps(rendererProps<ControlElement>());
+
+const input = useVanillaControl(
+  useJsonFormsControl(props),
+  (target) => target.value || undefined
+);
+const { styles, control, appliedOptions, onChange, isFocused } = input;
+
+defineOptions({
+  tester: rankWith(1, isStringControl) as RankedTester,
```

Comment:
I think we should call this `defaultTester`.

Reply:
In the current renderers it's called `tester` as we export the whole registry entry and there it must be named `tester`. However we already had use cases where people reused renderers with slightly different testers and/or ranks for the testers. So calling it the `defaultTester` made sense to me.

However I don't have any hard feelings here, so we could just keep `tester`